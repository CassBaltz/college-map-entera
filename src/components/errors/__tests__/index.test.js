import { shallow } from "enzyme";
import Errors from "..";

describe("Errors", () => {
  it("should render correctly - no errors", () => {
    const noErrors = [];
    const errors = shallow(<Errors errors={noErrors} />);
    expect(errors.getElement()).toBeNull();
  });

  it("should render correctly - errors exit", () => {
    const someErrors = [{ message: "Cannot perform that action" }];
    const errors = shallow(<Errors errors={someErrors} />);
    expect(errors.getElement()).toMatchSnapshot();
  });
});
