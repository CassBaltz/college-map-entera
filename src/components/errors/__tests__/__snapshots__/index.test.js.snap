// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Errors should render correctly - errors exit 1`] = `
<ul
  className="errorsWrapper"
>
  <li>
    Cannot perform that action
  </li>
</ul>
`;
